frontend/src/features/properties/map/shared/update/properties/UpdateProperties.tsx

```diff
@@ -4,29 +4,27 @@ import SidebarFooter from 'features/properties/map/shared/SidebarFooter';
 import MapSelectorContainer from 'features/properties/selector/MapSelectorContainer';
 import { IMapProperty } from 'features/properties/selector/models';
 import { FieldArray, Formik, FormikProps } from 'formik';
-import { Api_ResearchFile } from 'models/api/ResearchFile';
+import { Api_File } from 'models/api/File';
 import { useRef, useState } from 'react';
 import { Col, Row } from 'react-bootstrap';
 
-import { PropertyForm, ResearchForm } from '../../add/models';
-import { useUpdateResearchProperties } from '../../hooks/useUpdateResearchProperties';
+import { FileForm, PropertyForm } from '../../models';
 import { UpdatePropertiesYupSchema } from './UpdatePropertiesYupSchema';
 
-interface IUpdatePropertiesProps {
-  researchFile: Api_ResearchFile;
+export interface IUpdatePropertiesProps {
+  file: Api_File;
   setIsShowingPropertySelector: (isShowing: boolean) => void;
   onSuccess: () => void;
+  updateFileProperties: (file: Api_File) => Promise<Api_File | undefined>;
 }
 
 export const UpdateProperties: React.FunctionComponent<IUpdatePropertiesProps> = props => {
```

Comment:
This component can now operate on any file type by extracting the research specific embedded call to updateResearchFileProperties to a prop.